content/2017.03.building-projects-as-a-distributed-team.rst

```diff
@@ -0,0 +1,251 @@
+- Kinto: Building synchronization services with backup and sharing capability in mind.
+- RemoteServiceWorker: Building a Firefox Service Worker in the cloud
+- WebPush channels: Building a PubSub on top of WebPush
+
+While most of these projects have been built, deployed in production
+and eventually decommissined, some of them are still actively use by
+Firefox users everyday.
+
+
+How do you keep motivated building things that are thrown away?
+===============================================================
+
+Let's be honest, everything we build: cars, houses, and even softwares
+are meant to be eventually thrown away. That's why recycling is so
+important.
+
+The question is why do we keep building them nevertheless.
+
+My daily job is to help Mozilla make the web a safe place for users
+and prevent big companies to turn them into advertisement products
+without their consent.
+
+One way of doing so is to build software that empower people to use
+internet while being in charge of their own data.
+
+When users give their data to big companies, they are given them
+leverage, in the meantime these companies are offering a service that
+solve people problems, and they get addicted to it and come back for
+more.
+
+We alone cannot compete with companies that are spending millions of
+dollars building great products to capture users.
+
+However, we can build software that, in one hand, is useful for the
+Mozilla mission and makes us reactive to fix Mozilla issues while
+building Firefox, and in the other hand enpower developers to build
+great software where users choose where they want to store their data.
+
+What matters rather than building software is the vision that this
+software carries.
+
+Writting lines of code is cheap and fast. You can build a prototype in
+two weeks and iterate until you are happy with the result.
+
+Then it might be use by some people and it might get some traction for
+few months or few years and it will be eventually forgotten in a git
+repository somewhere forever.
+
+Writting software to solve problems is not enough, to keep motivated on the long term.
+
+What you need is a vision and make sure you work toward that vision.
+
+
+How do you turn a vision into OKR?
+==================================
+
+OKR means **Objectives and Key results**, it is a way to define
+tangible objectives and key results. You can then look at the results
+to tell if you were able to achieve your objectives.
+
+It is also a way to cut your vision in steps and to move your vision
+forward.
+
+OKR should be defined for quite short period of time to stay agile and
+be able to adapt to new environment constraints.
+
+At Mozilla we define quaterly OKR.
+
+
+How do you set course on a vision?
+==================================
+
+To be able to define OKR from your vision, you need to be able to find
+the fastest way to get closer to your vision, however what I've learn
+from sailing is that the fastest way is rarely the shortest.
```

Comment:
This section makes it sound like you have some really interesting ideas about how to choose OKRs according to speed vs. directness, but then you don't actually talk about how to set course.